src/ablog/tests/test_parallel.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+from subprocess import run
+
+
+def test_not_safe_for_parallel_read(rootdir: Path):
+    """Ablog is NOT safe for parallel read. In such case, it doesn't collect any posts."""
+    # https://github.com/sunpy/ablog/issues/297
+    # Very ugly hack to change the parallel_read_safe value to True
+    good_read_safe = '"parallel_read_safe": False'
+    bad_read_safe = '"parallel_read_safe": True'
+    init_py_path = Path(__file__).parent.parent / "__init__.py"
+    assert good_read_safe in init_py_path.read_text()
+    bad_init_py = init_py_path.read_text().replace(good_read_safe, bad_read_safe)
+    init_py_path.write_text(bad_init_py)
+
+    # liborjelinek: I wasn't able to demonstrate the issue with the `parallel` argument to the `sphinx` fixture
+    # @pytest.mark.sphinx("html", testroot="parallel", parallel=2)
+    # therefore running sphinx-build externally
+    indir = rootdir / "test-parallel"
+    run(["sphinx-build", "-M", "html", indir.as_posix(), "_build", "-j", "auto"], check=True)
+
+    outdir = indir / "_build" / "html"
+    assert (outdir / "postlist.html").exists()
```

Comment:
```
=========================== short test summary info ============================
FAILED .tox/py312-sphinx8/lib/python3.12/site-packages/ablog/tests/test_parallel.py::test_not_safe_for_parallel_read - AssertionError: assert False
 +  where False = exists()
 +    where exists = (PosixPath('/home/runner/work/ablog/ablog/roots/test-parallel/_build/html') / 'postlist.html').exists
```
This file doesn't seem to exist when run on github actions.

Reply:
You're right. I have tested it many times locally. It looks that GitHub Actions runner have some security limits to read files created... I'll try to consult the docs.

Reply:
A note to myself: Yes, GH Actions really somehow prevent to access created files. I tried `sudo ls -l` of a file produced by test and it
```
ls: cannot access '/home/runner/work/ablog/ablog/roots/test-parallel/_build/html/postlists.html': No such file or directory
```

Reply:
I wonder if it's possible to use pytest's tmpdir fixture to write the output ablog files there?

Reply:
Okay, it was a the stupidest error ever :-) The created file is `postlist.html`, not `postlists.html`. However, I've kept building into Pytest temp dir. Good idea anyway. Test is passing now.

Reply:
Thanks!